### PR TITLE
Remove the functional tests run from our Jenkinsfile

### PR DESCRIPTION
**Pull request by willbarton**
Because the functional tests have been so temperamental running against the Docker stacks, we're going to temporarily remove them from the PR cycle and run them in our continuous deploy to staging.

This change removes everything related to them from the Jenkinsfile.

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -33,9 +30,7 @@ pipeline {
         DEPLOY_SUCCESS = false
         // Determines if Elasticsearch image should be updated
         IS_ES_IMAGE_UPDATED = 'false'
-        // Determines if Cypress image should be updated
-        IS_CYPRESS_IMAGE_UPDATED = 'false'
-        DOCKER_HUB_REGISTRY = 'https://hub.docker.com'
+        DOCKER_HUB_REGISTRY = 'https://registry.hub.docker.com'
```
Comment (chosak) [inline Jenkinsfile:33]: I think perhaps we can remove this definition and use the default `docker login` below without a registry parameter.

File: Jenkinsfile [hunk 2]
```diff
@@ -129,7 +114,7 @@ pipeline {
                             usernameVariable: 'DOCKER_HUB_USER'
                         )
                     ]) {
-                        sh 'docker login -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD'
+                        sh 'docker login $DOCKER_HUB_REGISTRY -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD'
```
Comment (chosak) [inline Jenkinsfile:117]: See above; will this work if we don't provide the registry URL?

**Review by chosak — APPROVED** [review]
✂️ ✂️ ✂️ ✂️ ✂️ ✂️ ✂️ ✂️ ✂️ ✂️ ✂️

**Review by higs4281 — APPROVED** [review]
Thank you for your service, Cypress

File: Jenkinsfile (hunk 1 above)
Comment (willbarton) [reply Jenkinsfile:33]: That doesn't work — see the build history of https://github.com/cfpb/consumerfinance.gov/pull/6498#partial-pull-merging

File: Jenkinsfile (hunk 2 above)
Comment (willbarton) [reply Jenkinsfile:117]: See comment above.

**Pull request merged**